python/src/aiconfig/editor/client/src/components/prompt/PromptContainer.tsx

```diff
@@ -38,6 +38,25 @@ type Props = {
 
 export const PROMPT_CONTAINER_HEIGHT_MAP = new Map<string, number>();
 
+// Set default styles, which can be overridden by the provided themeOverride
+// or theme associated with mode
+const useStyles = createStyles((theme) => ({
+  cellStyle: {
+    borderBottomRightRadius: "0px",
+    borderTopRightRadius: "0px",
+    flex: 1,
+  },
+
+  sidePanel: {
+    border: `1px solid ${
+      theme.colorScheme === "dark" ? theme.colors.dark[4] : theme.colors.gray[3]
+    }`,
+    borderBottomRightRadius: "4px",
+    borderLeft: "none",
+    borderTopRightRadius: "4px",
```

Comment:
These values are obtained directly from existing Themes for the modes and are identical in each